### DISCUSSION TITLE
Change opacity of view

### DISCUSSION THREAD

**Discussion by Miao-10k** [google/filament#6237] [General]
I am using filament on Android for an AR scene. I want to reduce the opacity of the 3d scene render result when the camera getting close to it. Is there any way I can change the opacity of the view or set opacity with postprocessing ？Also, I have tried to implement this by setting material of the scene to TRANSPARENT or FADE and use TWO_PASSES_ONE_SIDE or TWO_PASSES_TWO_SIDES which cannot meet our requirements since some undesired see-through situation.

**Comment by romainguy** [comment]
You'd have to render the 3D scene into a texture to do this. Alternatively you could render Filament and the camera feed as separate widgets and fade filament at the os level.

**Reply by Miao-10k** [reply]
Thanks! Is there any sample on how to render a scene into a texture?
